server/src/main/java/org/kaazing/gateway/server/config/parse/GatewayConfigParser.java

```diff
@@ -158,16 +159,30 @@ private File getTranslatedConfigFile(final File configFile)
         Document dom = xmlReader.build(configFile);
         Element root = dom.getRootElement();
         GatewayConfigNamespace namespace =  GatewayConfigNamespace.fromURI(root.getNamespace().getURI());
-
+        checkForOutdated(root); 
         boolean writeTranslatedFile = !namespace.equals(GatewayConfigNamespace.CURRENT_NS);
         File translatedConfigFile = writeTranslatedFile ?
                 new File(configFile.getParent(), configFile.getName()
                 + TRANSLATED_CONFIG_FILE_EXT) : configFile;
 
         translate(namespace, dom, translatedConfigFile, writeTranslatedFile);
-
         return translatedConfigFile;
     }
+    
+    private void checkForOutdated(Element root) throws Exception {
+        Namespace namespace = root.getNamespace();
+        List<Element> children = root.getChildren("service", namespace);
+        for (Element child : children) {
+            Element typeChild = child.getChild("type", namespace);
+            String type = typeChild.getText();
+            if (type.equals("management.snmp")) {
+                throw new Exception("snmp management type is outdated and no longer supported."); 
```

Comment:
change from outdated to just "is no longer supported", suggest renaming method name accordingly